src/dog.sol

```diff
@@ -150,7 +150,7 @@ contract Dog {
     function chop(bytes32 ilk) external view returns (uint256) { return ilks[ilk].chop; }
 
     // --- CDP Liquidation: all bark and no bite ---
-    function bark(bytes32 ilk, address urn) external returns (uint256 id) {
+    function bark(bytes32 ilk, address urn, address guy) public returns (uint256 id) {
```

Comment:
I think we should use `usr` which is the standard now.

Reply:
fixed

Reply:
I think you missed to push the change or something is not getting updated.